setup.py

```diff
@@ -12,87 +12,86 @@
 
 from setuptools import find_packages, setup
 
-readme = open('README.rst').read()
-history = open('CHANGES.rst').read()
+readme = open("README.rst").read()
+history = open("CHANGES.rst").read()
 
 tests_require = [
-    'pytest-invenio>=1.4.0',
+    "pytest-invenio>=1.4.0",
 ]
 
 extras_require = {
-    'docs': [
-        'Sphinx>=1.5.1',
+    "docs": [
+        "Sphinx>=1.5.1",
     ],
-    'tests': tests_require,
+    "tests": tests_require,
 }
 
-extras_require['all'] = []
+extras_require["all"] = []
 for reqs in extras_require.values():
-    extras_require['all'].extend(reqs)
+    extras_require["all"].extend(reqs)
 
 setup_requires = [
-    'Babel>=1.3',
-    'pytest-runner>=3.0.0,<5',
+    "Babel>=1.3",
+    "pytest-runner>=3.0.0,<5",
 ]
 
 install_requires = [
-    'Flask-BabelEx>=0.9.4',
-    'elasticsearch_dsl>=7.2.1',
-    'invenio-rdm-records~=0.18.3',
-    'invenio_search>=1.3.1',
-
+    "Flask-BabelEx>=0.9.4",
+    "elasticsearch_dsl>=7.2.1",
+    "invenio-rdm-records~=0.18.3",
```

Comment:
```suggestion
    "invenio-rdm-records~=0.20.8",
```

Reply:
in my opinion it is not the task of this PR to make this change. this pull request is about to move to black. this should be done in another pull request.